### ISSUE TITLE
Turn off automatic generated properties of polymorphic subtype

### ISSUE THREAD

**Issue by bantolov** [Rhetos/Rhetos#62]
Currently, the properties of the polymorphic are automatically **copied** to the subtype, if an explicit mapping is not defined. For example, the following snippet from [documentation](https://github.com/Rhetos/Rhetos/wiki/Polymorphic-concept) will generated the `EventDate` property on the `Entity BorrowMoney`.

Unfortunately this behavior makes it easy for developers to forget to implement the subtype property and generates bug with empty subtype properties with null values.

```
Module Demo
{
    Polymorphic MoneyTransaction
    {
        DateTime EventDate;
        Money Amount;
    }

    Entity BorrowMoney
    {
        ShortString FromWhom;
        Money Amount;

        Is Demo.MoneyTransaction;
        // The EventDate will be automatically added. The Amount is automatically mapped.
    }
```

**Expected behavior**
Deploying the code above should result with a DslSyntaxException exception "The property Money Demo.MoneyTransaction.Amount is not implemented in the polymorphic subtype Entity Demo.BorrowMoney. Please add the property implementation to the subtype.".

The working script should look like this:

```
    Entity BorrowMoney
    {
        ShortString FromWhom;
        DateTime EventDate;
        Money Amount;

        Is Demo.MoneyTransaction;
       // The EventDate and Amount are automatically mapped.
    }

```

**Notes**

* To allow for backward compatibility, the automatic creation of properties should be left in the code, activated by the bool switch in web.config appSettings: "CommonConcepts.Legacy.AutoGeneratePolymorphicProperty". This switch should be added to the web.config with the value "False", but when reading it the default value should be True (if missing), so that the existing applications use the old behavior.
* Update the documentation.

**Comment by ibarban** [comment]
I was trying to implement a test before implementing this requirement and run into the the issue that SqlUtility is a static class and the method Identifier requires that the Rhetos server is set up.
The problem is that I wanted to implement a unit test for this and I would need a mock of the SqlUtility.
So my question is if there is a reason why this is a static class and not an interface implementation?

**Comment by bantolov** [comment]
SqlUtility should definitely be refactored to dependency-injection singleton, instead of the current static class. The ISqlUtility interface is already created for that purpose, but there are a lot of members left to migrate from SqlUtility to ISqlUtility.

Since this feature is already partially tested on each deployment of CommonConceptsTest and by the existing unit tests in PolymorphicTest class, in this case I would recommend to avoid writing the unit test.

**Comment by bantolov** [comment]
Closed by #65
